### ISSUE TITLE
Year addition perform unexpected result when value is a multiple of 5

### ISSUE THREAD

**Issue by YixiongJiang** [spencermountain/spacetime#285]
```js
const date = spacetime('2000-01-01 00:00:00');
expect(date.add(30, 'year').format(DateFormat.YYYY_MM_DD_HH_MM_SS)).toBe('2030-01-01 00:00:00');
```
Expected: "2030-01-01 00:00:00"
Received: "2029-01-01 00:00:00"

It happens when unit is year and when the value is a multiple of 5.

Just curiosity, why not just add value to the year numerically?

**Comment by spencermountain** [comment]
thanks @YixiongJiang good find. I'll check it out.
cheers

**Comment by YixiongJiang** [comment]
@spencermountain Thanks, for your more information, I found there is a workaround to avoid this bug:
date.add(30*12, 'month'),
will get correct result: "2030-01-01 00:00:00"

**Comment by spencermountain** [comment]
yikes! will add this to the next release, ideally next week.
thank you

**Comment by spencermountain** [comment]
fixed in `v6.16.3` - thank you!
